cmd/release.go

```diff
@@ -240,11 +241,17 @@ func getChangelog(version string, rc io.ReadCloser) (string, string) {
 		s       []string
 		header  string
 		reading bool
+
+		releaseHeaderPattern = "## " + strings.ReplaceAll(version, ".", "\\.") + " / \\d{4}-\\d{2}-\\d{2}"
 	)
 	for (len(s) == 0 || reading) && scanner.Scan() {
 		text := scanner.Text()
 		switch {
-		case strings.HasPrefix(text, "## "+version+" "):
+		case strings.HasPrefix(text, "## "+version):
+			if valid, _ := regexp.MatchString(releaseHeaderPattern, text); !valid {
+				fatal(errors.New("Found invalid release header in changelog for version '" + version + "'.  " +
```

Comment:
please return this error instead of `panicking/io.Exit(1)` in random place. Ideally the rule of thumb is that io.Exit/panic should be only in `func main()`. So we could start having good practice here (: 

What do you think?

Reply:
Sounds good, I think `runRelease` is kind of a main for the `release` command, so should we keep the calls to `panic` there, or should the error bubble up further?

Reply:
It should bubble, but let's start simple in this PR, I am fine with `runRelease` exits only for now. (: We can improve later.